Example/Tests/CDTQQuerySqlTranslatorTests.m

```diff
@@ -696,31 +699,37 @@
                     @{ @"age" : @{@"$eq" : @12} },
                     @{ @"pet" : @{@"$not" : @{@"$eq" : @"cat"}} }
                 ];
-                CDTQSqlParts *parts = [CDTQQuerySqlTranslator wherePartsForAndClause:query];
+                CDTQSqlParts *parts = [CDTQQuerySqlTranslator wherePartsForAndClause:query
+                                                                          usingIndex:@"named"];
+                NSString *tableName = [CDTQIndexManager tableNameForIndex:@"named"];
+                NSString *select = [NSString stringWithFormat:@"SELECT _id FROM %@", tableName];
+                NSString *whereName = @"WHERE \"name\" = ?";
+                NSString *wherePet = @"WHERE \"pet\" = ?";
+                NSString *expectedName = [NSString stringWithFormat:@"_id NOT IN (%@ %@)",
+                                          select, whereName];
+                NSString *expectedAge = @"\"age\" = ?";
+                NSString *expectedPet = [NSString stringWithFormat:@"_id NOT IN (%@ %@)",
+                                         select, wherePet];
+                NSString *expected = [NSString stringWithFormat:@"%@ AND %@ AND %@",
+                                      expectedName, expectedAge, expectedPet];
```

Comment:
I'd rather see the whole string here rather than building it up in code. Particularly using things like `tableNameForIndex` is asking for trouble because the behaviour of that method could change in a regressive way and the test wouldn't pick it up.